jena-arq/src/main/java/org/apache/jena/sparql/engine/QueryIterator.java

```diff
@@ -18,15 +18,13 @@
 
 package org.apache.jena.sparql.engine;
 
-import java.util.Iterator ;
-
-import org.apache.jena.atlas.lib.Closeable ;
 import org.apache.jena.sparql.engine.binding.Binding ;
 import org.apache.jena.sparql.util.PrintSerializable ;
+import org.apache.jena.util.iterator.ClosableIterator;
 
 /** Root of query iterators in ARQ. */
 
-public interface QueryIterator extends Closeable, Iterator<Binding>, PrintSerializable
+public interface QueryIterator extends ClosableIterator<Binding>, PrintSerializable
```

Comment:
This change isn't necessary but if it is made, please use `IteratorCloseable`  See the comments in ClosableIterator which refer to the Model API and `ExtendedIterator` - let's keep `ClosableIterator` for jena-core.